internal/names/names.go

```diff
@@ -0,0 +1,14 @@
+package names
+
+const (
+	// Attributes for the terraform resources.
+	AttrBucket     = "bucket"
+	AttrAcl        = "acl"
+	AttrDomainName = "domain_name"
+
+	// Headers for the requests to Tigiris.
+	HeaderContentType   = "Content-Type"
+	HeaderAccept        = "Accept"
+	HeaderAmzContentSha = "X-Amz-Content-Sha256"
+	HeaderAmzIdentityId = "S3-Identity-Id"
```

Comment:
We don't need to expose it.

Reply:
Do you mean not export it from the package?

Reply:
Updated

Reply:
I mean we can completely remove it once we fix it in Gateway.

Reply:
Yeah I will remove it then.